charts/consul/values.yaml

```
@@ -1859,6 +1859,85 @@ connectInject:
     # Note: This value has no effect if transparent proxy is disabled on the pod.
     defaultOverwriteProbes: true
 
+  # Configures consul-cni plugin for Consul Service mesh services
+  # Using this feature requires Consul 1.13.0-beta+
+  cni:
+    # If true, then all traffic redirection setup with use the consul-cni plugin
+    # @type: boolean
+    enabled: false
+
+    # Namespace to install cni plugin.
+    # @type: string
+    namespace: kube-system
+ 
+    # If multus plugin is enabled or not. If true, the cni installer will generate a consul cni configuration
```

nit: We should use proper capitalization everywhere since these are used to generate docs.
```suggestion
    # If multus plugin is enabled or not. If true, the CNI installer will generate a consul CNI configuration
```

---

Thank you!  I have updated them to be capitalized unless they were mentioning a specific setting  (ie cniBinDir) or a binary (ie consul-cni)